.github/workflows/deploy.yaml

```diff
@@ -31,6 +31,7 @@ jobs:
           test_submission: true
           package_name: MDAnalysisData
           module_name: 'MDAnalysisData'
+          test_deps: 'pytest pytest-mock'
```

Comment:
Note: This information is effectively duplicated from the information in setup.py.

Reply:
Unfortunately how this is stored in setup.py isn't consistent, even across our own projects - lack of enforced guidelines makes these things necessary.

A good example of this is `doc` vs `docs` (which has driven me crazy lately).

Reply:
I believe you!